tests/playwright/src/model/pages/resources-kind-page.ts

```diff
@@ -0,0 +1,55 @@
+/**********************************************************************
+ * Copyright (C) 2024 Red Hat, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ ***********************************************************************/
+
+import { expect as playExpect, type Locator, type Page } from '@playwright/test';
+
+import type { KindClusterOperations } from '../core/kind-cluster-operations';
+import { CreateKindClusterPage } from './create-kind-cluster-page';
+import { ResourcesPage } from './resources-page';
+
+export class ResourcesKindCard extends ResourcesPage {
```

Comment:
you should not inherit from resourcesPage since you cannot really use the locator from there in here. Just use BasePage. example: `src/model/pages/extension-card-page`.

Reply:
And make it please universal, since any other connection type can be used, add parameter for constructor.

Reply:
Maybe, the best would be if you create a follow up issue, where you will refactor the Resource Card Pages. Ie.:
Abstract ResourceCardPage and two classes that inherits from it, ResourceConnectionCardPage (podman, kind, etc.) and CliResourceCardPage (kubectl, compose).

Reply:
https://github.com/containers/podman-desktop/pull/8310